depobs/website/config.py

```diff
@@ -119,6 +119,15 @@
     save_to_db=True,
 )
 
+HIBP_CLIENT = {
+    **_aiohttp_args,
+    **dict(
+        base_url=os.environ.get("HIBP_BASE_URL", "https://haveibeenpwned.com/api/v3/"),
+        additional_headers={"hibp-api-key": os.environ.get("HIBP_AUTH_TOKEN", None)},
+        max_connections=1,
+        user_agent="dependency_observatory",  # Don't know why the user_agent above doesn't work
```

Comment:
weird that it's not using the default `_aiohttp_args`

nit: can we set this to `_aiohttp_args["user_agent"]` so it's defined in one place?